providers/celery/pyproject.toml

```diff
@@ -61,7 +61,8 @@ dependencies = [
     # The Celery is known to introduce problems when upgraded to a MAJOR version. Airflow Core
     # Uses Celery for CeleryExecutor, and we also know that Kubernetes Python client follows SemVer
     # (https://docs.celeryq.dev/en/stable/contributing.html?highlight=semver#versions).
-    "celery[redis]>=5.4.0,<6",
+    # Updating min version to 5.5.0 to avoid redis import bug https://github.com/apache/airflow/issues/41359
+    "celery[redis]>=5.5.0,<6",
```

Comment:
Leaving comment to not merge it accidentally without testing manually with CeleryExecutor

Reply:
`breeze testing core-integration-tests --integration celery`
passed on the celery branch
![Screenshot 2025-04-01 at 14 44 09](https://github.com/user-attachments/assets/07390c8e-5283-4b4e-b29f-36e64f0be39f)